### PR TITLE
Fix BUCK build after #103185

### PR DESCRIPTION
**Pull request by huydhn**
Per title.

### <samp>🤖 Generated by Copilot at c64f0c0</samp>

> _`torch_headers` grows_
> _to include profiler files_
> _autumn of code change_

### CODE REVIEW

**Review by atalman — APPROVED** [review]

**Review by kit1980 — APPROVED** [review]

**Review by malfet — APPROVED** [review]

File: buckbuild.bzl [hunk 1]
```diff
@@ -1998,6 +1998,8 @@ def define_buck_targets(
                 ("", "torch/csrc/jit/python/*.h"),
                 ("", "torch/csrc/jit/frontend/*.h"),
                 ("", "torch/csrc/jit/serialization/*.h"),
+                ("", "torch/csrc/profiler/*.h"),
```
[the remark this answers is not in the document]
Comment (malfet) [reply buckbuild.bzl]: I think buck1 vs buck2 has some reservation about recursive glob

[the remark this answers is not in the document]
Comment (kit1980) [reply buckbuild.bzl]: Should be fine, there are already ** globs in this file.

**Pull request closed without merging**